internal/controllers/synthesis/exec.go

```diff
@@ -20,8 +20,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-// maxSliceJsonBytes is the max sum of a resource slice's manifests. It's set to 1mb, which leaves 512kb of space for the resource's status, encoding overhead, etc.
-const maxSliceJsonBytes = 1024 * 768
+// maxSliceJsonBytes is the max sum of a resource slice's manifests.
+const maxSliceJsonBytes = 1024 * 512
```

Comment:
I caught a few cases where we got some errors back from etcd because resource slices were too large. This may need to be refined more in the future

Reply:
Interesting, I guess the status grows linearly, do you think that was causing the issue?